sdk/anomalydetector/Azure.AI.AnomalyDetector/src/GlobalSuppressions.cs

```diff
@@ -4,5 +4,4 @@
 using System.Diagnostics.CodeAnalysis;
 
 [assembly: SuppressMessage("Usage", "AZC0007: A client type should have a public constructor with equivalent parameters that doesn't take a Azure.Core.ClientOptions-derived type as the last argument", Justification = "")]
-[assembly: SuppressMessage("Usage", "AZC0006: A client type should have a public constructor with equivalent parameters that takes a Azure.Core.ClientOptions-derived type as the last argument", Justification = "")]
-[assembly: SuppressMessage("Usage", "AZC0002: Client method should have cancellationToken as the last optional parameter (both name and it being optional matters", Justification = "")]
\ No newline at end of file
+[assembly: SuppressMessage("Usage", "AZC0006: A client type should have a public constructor with equivalent parameters that takes a Azure.Core.ClientOptions-derived type as the last argument", Justification = "")]
```

Comment:
Please use individual `#pragma`s and not _GlobalSuppressions.cs_. It's too broad and hides other instances.

Reply:
I'm not adding this. It exists before, which is irrelevant to this PR. Let's handle it in another PR since our customers are waiting for this.